lib/plugins/inspec-init/lib/inspec-init/templates/profile_gcp/attributes.yml

```diff
@@ -0,0 +1 @@
+# gcp_project_id: 'your-gcp-project'
```

Comment:
Since this is commented, can we remove it?

Reply:
Hi @jerryaldrichiii, this was deliberately left to make life easier for the consumer, I've added a line to better reflect this in the latest update.